### DISCUSSION TITLE
Small spelling/grammar/typo fixes

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#3497] [Repository Maintenance]
Many folks here are not native speakers, and infrequently-but-regularly make minor spelling/grammar mistakes or typos. For example, @westnordost sometimes typos *is* as *ist* (probably from muscle memory— it's the same word in German, his native language).

Occasionally, I have just gone ahead and edited your comments to fix these— **only** when I'm ~100% certain that I am not changing your intent *or* the phrasing (ie. your "voice"). For example (which prompted me to write this), I changing https://github.com/streetcomplete/StreetComplete/pull/3495#issuecomment-964429230 to "Sounds useful". So, it's not a big deal one way or another, but I figure I should still ask: Would you like me to continue doing this in the future?

**Comment by HolgerJeromin** [comment]
I would not change other peoples Github comments. 
I do so only in issue titles and in issue descriptions when the content is not clear without thinking a bit (or searchable).

**Comment by westnordost** [comment]
See Holger. Please do not change other people's comments. There are also better ways to spend oneselves time anyway.

**Reply by smichel17** [reply]
It does not take much time, when I have the page open already. They do not require any time thinking, either, because if I had to think about it, then it means the meaning is not 100% obvious and thus I should not edit it.
Example: I would have `s/hte/the/` in  https://github.com/streetcomplete/StreetComplete/issues/3529#issuecomment-977783063. Takes 5 seconds (much shorter than writing my comment in that thread).

**Reply by mnalis** [reply]
@smichel17 I still don't see much benefit even for obvious typos in issues as such. Now, if it was something that could be made clearer in the code or documentation, then yes, it should be fixed, but for the one-time comment which is trivially readable, not only is it not worth it, but it is IMHO often counterproductive (because other people would then click on the edit history to see what changed, and we don't even have "minor" edit distinction on GitHub which could be hidden (like one has Mediawiki for example)...

IMHO, if it is _trivially_ understandable typo, it should be left borken as it is. And if it is _not_ trivially understandable, then then the author should be asked for clarification instead. Also note that (having a pressure of) knowing there is grammar police breathing down one's neck is usually not enjoyable experience, either :smile:. It might even be considered rude by some people (esp. depending on cultural differences)...

**Reply by smichel17** [reply]
Fair enough (although the intention is to improve readability while *avoiding* bothering people about fixing trivial mistakes — more "reliable autocorrect" than grammar police).

**Comment by matkoniecz** [comment]
The same, if I keep making some typo/mistake feel free to let me know - using email is fine (people let me know about whatever/whether and nowadays I think I am better with it).

I would reserve editing for cases like

- obviously unwanted content present there as result of copy paste, especially private one (but edit history is present!)
- abusive content
- technical issues with markdown syntax, maybe - I am not fully sure here
